### PR TITLE
[Docs] PR for Update Kubernetes versions for EKS and GKE and impove content

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md [hunk 1]
```diff
@@ -134,22 +140,22 @@ Table depicting acceptable port names, applicable component (Master/TServer) and
 
 #### podManagementPolicy
 
-Specify pod management policy for Statefulsets created as part of YugabyteDB cluster. Valid values are `Parallel` & `OrderedReady`, `Parallel` being the default value.
+Specify pod management policy for Statefulsets created as part of a YugabyteDB cluster. Valid values are `Parallel` and `OrderedReady`, with `Parallel` being the default value.
 
 #### storage
 
-Specify storage configurations viz. Storage `count`, `size` & `storageClass` of volumes. Typically, 1 volume per Master instance is sufficient, hence Master has a default storage count of `1`. If storage class isn't specified, it will be defaulted to `standard`. Make sure kubernetes admin has defined `standard` storage class, before leaving this field out.
+Specify storage configurations viz. Storage `count`, `size`, and `storageClass` of volumes. Typically, 1 volume per YB-Master instance is sufficient, hence YB-Master has a default storage count of `1`. If the storage class is not specified, it defaults to `standard`. Make sure the Kubernetes admin has defined the`standard` storage class, before leaving this field out.
```
Comment (ddhodge) [inline docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md:147]: Specify storage configurations, namely storage

File: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md [hunk 2]
```diff
@@ -134,22 +140,22 @@ Table depicting acceptable port names, applicable component (Master/TServer) and
...
 
 #### resources
 
-Specify resource `requests` and `limits` under `resources` attribute. The resources to be specified are `cpu` and `memory`. The `resource` property in itself is optional and will not be applied to created `StatefulSets`, if omitted. You may also choose to specify either `resource.requests` or `resource.limits`, or both.
+Specify resource `requests` and `limits` under the `resources` attribute. The resources to be specified are `cpu` and `memory`. The `resource` property in itself is optional and is not applied to created `StatefulSets`, if omitted. You may also specify either `resource.requests` or `resource.limits`, or both.
 
 #### flags
 
-Specify flags for additional control of the YugabyteDB cluster. For available, see [YB-Master flags](../../../../../reference/configuration/yb-master/#flags) and [YB-TServer flags](../../../../../reference/configuration/yb-tserver/#flags).
+Specify flags for additional control of the YugabyteDB cluster. For for information, see [YB-Master flags](../../../../../reference/configuration/yb-master/#flags) and [YB-TServer flags](../../../../../reference/configuration/yb-tserver/#flags).
```
Comment (ddhodge) [inline docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md:155]: For more information

**Review by ddhodge — APPROVED** [review]
misc comments

File: docs/content/preview/deploy/kubernetes/best-practices.md [hunk 3]
```diff
@@ -63,4 +63,6 @@ Below is a table that summarizes the features and when to use local or remote st
   </tr>
 </table>
 
-Thus, it is generally preferable to use local storage where possible for higher performance and lower costs. The [GKE section](../gke/statefulset-yaml/) shows how to deploy YugabyteDB on Kubernetes using local SSDs.
+
+It is generally preferable to use local storage where possible for higher performance and lower costs. The [GKE section](../gke/statefulset-yaml/) shows how to deploy YugabyteDB on Kubernetes using local SSDs.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/best-practices.md:67]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 4]
```diff
@@ -11,49 +11,56 @@ menu:
...
 NAME          TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                                        AGE
 yb-tservers   ClusterIP   None         <none>        7100/TCP,9000/TCP,6379/TCP,9042/TCP,5433/TCP   56m
 ```
 
 The following example shows a client that uses the YSQL shell ([`ysqlsh`](../../../admin/ysqlsh/)) to connect:
 
 ```sh
-$ kubectl run ysqlsh-client -it --rm  --image yugabytedb/yugabyte-client --command -- ysqlsh -h yb-tservers.yb-demo.svc.cluster.local
+kubectl run ysqlsh-client -it --rm  --image yugabytedb/yugabyte-client --command -- ysqlsh -h yb-tservers.yb-demo.svc.cluster.local
 yugabyte=# CREATE TABLE demo(id INT PRIMARY KEY);
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:37]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 5]
```diff
@@ -11,49 +11,56 @@ menu:
...
+```output
 CREATE TABLE
 ```
 
 The following example shows a client that uses the YCQL shell ([`ycqlsh`](../../../admin/cqlsh)) to connect:
 
 ```sh
-$ kubectl run cqlsh-shell -it --rm  --image yugabytedb/yugabyte-client --command -- cqlsh yb-tservers.yb-demo.svc.cluster.local 9042
+kubectl run cqlsh-shell -it --rm  --image yugabytedb/yugabyte-client --command -- cqlsh yb-tservers.yb-demo.svc.cluster.local 9042
 ycqlsh> CREATE KEYSPACE demo;
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:48]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 6]
```diff
@@ -11,49 +11,56 @@ menu:
...
 ycqlsh> use demo;
 ycqlsh:demo> CREATE TABLE t_demo(id INT PRIMARY KEY);
 ```
 
 Note that although tables are [internally sharded](../../../architecture/concepts/yb-tserver/) across multiple tserver pods, every tserver pod has the ability to process any query, irrespective of its actual tablet assignment.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:53]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 7]
```diff
@@ -11,49 +11,56 @@ menu:
...
 
-## Connecting Externally
+## Connecting externally
 
-An application that is deployed outside the Kubernetes cluster should use the external LoadBalancer IP address to connect to the cluster. Connections to the load balancer IP address are randomly routed to one of the tserver pods behind the yb-tservers service.
+An application that is deployed outside the Kubernetes cluster should use the external LoadBalancer IP address to connect to the cluster. Connections to the load balancer IP address are randomly routed to one of the tserver pods behind the yYB-TServer service:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:57]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 8]
```diff
@@ -11,49 +11,56 @@ menu:
...
 
 ```sh
-$ kubectl get svc -n yb-demo
+kubectl get svc -n yb-demo
+```
+
+```output
 NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                                        AGE
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:64]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 9]
```diff
@@ -11,49 +11,56 @@ menu:
...
-$ kubectl run ysqlsh-client -it --rm  --image yugabytedb/yugabyte-client --command -- ysqlsh -h yb-tservers.yb-demo.svc.cluster.local
+kubectl run ysqlsh-client -it --rm  --image yugabytedb/yugabyte-client --command -- ysqlsh -h yb-tservers.yb-demo.svc.cluster.local
 yugabyte=# CREATE TABLE demo(id INT PRIMARY KEY);
+```
+
+```output
 CREATE TABLE
 ```
 
 The following example shows a client that uses the YCQL shell ([`ycqlsh`](../../../admin/cqlsh)) to connect:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:44]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 10]
```diff
@@ -64,8 +71,11 @@ yb-tservers          ClusterIP      None            <none>        7100/TCP,9000/
 The following example shows a client that uses the YSQL shell ([`ysqlsh`](../../../admin/ysqlsh/)) to connect:
 
 ```sh
-$ docker run yugabytedb/yugabyte-client ysqlsh -h 98.138.219.232
+docker run yugabytedb/yugabyte-client ysqlsh -h 98.138.219.232
 yugabyte=# CREATE TABLE demo(id INT PRIMARY KEY);
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:75]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 11]
```diff
@@ -64,8 +71,11 @@ yb-tservers          ClusterIP      None            <none>        7100/TCP,9000/
...
+```
+
+```output
 CREATE TABLE
 ```
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:81]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 12]
```diff
@@ -11,49 +11,56 @@ menu:
...
+kubectl run cqlsh-shell -it --rm  --image yugabytedb/yugabyte-client --command -- cqlsh yb-tservers.yb-demo.svc.cluster.local 9042
 ycqlsh> CREATE KEYSPACE demo;
 ycqlsh> use demo;
 ycqlsh:demo> CREATE TABLE t_demo(id INT PRIMARY KEY);
 ```
 
 Note that although tables are [internally sharded](../../../architecture/concepts/yb-tserver/) across multiple tserver pods, every tserver pod has the ability to process any query, irrespective of its actual tablet assignment.
 
-## Connecting Externally
+## Connecting externally
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:55]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 13]
```diff
@@ -108,15 +124,17 @@ Status:
...
 To start a YugabyteDB cluster with encryption in transit (TLS) enabled, follow the steps at [Google Kubernetes Service (GKE) - Helm Chart](/preview/deploy/kubernetes/single-zone/gke/helm-chart/) and set the flag `tls.enabled=true` in the helm command line, as shown in the following example:
 
 ```shell
 helm install yugabyte --namespace yb-demo --name yb-demo --set=tls.enabled=true
 ```
 
-### Connect from Within the Kubernetes Cluster
+### Connect from within the Kubernetes cluster
 
 Copy the following `yb-client.yaml` and use the `kubectl create -f yb-client.yaml` command to create a pod with auto-mounted client certificates, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:139]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 14]
```diff
@@ -146,20 +164,31 @@ spec:
...
+kubectl exec -n yb-demo -it yb-client -- ysqlsh -h yb-tservers.yb-demo.svc.cluster.local "sslmode=require"
+```
+
+```output
 ysqlsh (11.2-YB-2.1.5.0-b0)
 SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
 Type "help" for help.
+```
 
+```sh
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:176]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 15]
```diff
@@ -146,20 +164,31 @@ spec:
...
 yugabyte=# \conninfo
+```
+
+```output
 You are connected to database "yugabyte" as user "yugabyte" on host "yb-tservers.yb-demo.svc.cluster.local" at port "5433".
 SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
 ```
 
 When a client uses the `YCQL shell` ([`ycqlsh`](../../../admin/cqlsh)) to connect, you can execute the following command to verify the connection:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:185]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 16]
```diff
@@ -146,20 +164,31 @@ spec:
...
 
 ```sh
-$ kubectl exec -n yb-demo -it yb-client -- ycqlsh yb-tservers.yb-demo.svc.cluster.local 9042 --ssl
+kubectl exec -n yb-demo -it yb-client -- ycqlsh yb-tservers.yb-demo.svc.cluster.local 9042 --ssl
+```
+
+```output
 Connected to local cluster at yb-tservers.yb-demo.svc.cluster.local:9042.
 [cqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
 Use HELP for help.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:194]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 17]
```diff
@@ -170,28 +199,39 @@ Connected to local cluster at yb-tservers.yb-demo.svc.cluster.local:9042.
...
+docker run -it --rm -v $(pwd)/certs/:/root/.yugabytedb/:ro yugabytedb/yugabyte-client:latest ysqlsh -h <External_Cluster_IP> "sslmode=require"
+```
+
+```output
 ysqlsh (11.2-YB-2.1.5.0-b0)
 SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
 Type "help" for help.
+```
 
+```sh
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:230]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 18]
```diff
@@ -170,28 +199,39 @@ Connected to local cluster at yb-tservers.yb-demo.svc.cluster.local:9042.
...
 yugabyte=# \conninfo
+```
+
+```output
 You are connected to database "yugabyte" as user "yugabyte" on host "35.200.205.208" at port "5433".
 SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
 ```
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:237]: fixed

File: docs/content/preview/deploy/kubernetes/clients.md [hunk 19]
```diff
@@ -201,10 +241,14 @@ When a client uses the `YCQL shell` ([`ycqlsh`](../../../admin/cqlsh)) to connec
...
 $ docker run -it --rm -v $(pwd)/certs/:/root/.yugabytedb/:ro \
 --env SSL_CERTFILE=/root/.yugabytedb/root.crt yugabytedb/yugabyte-client:latest ycqlsh <External_Cluster_IP> 9042 --ssl
+```
+
+```output
 ysqlsh (11.2-YB-2.1.5.0-b0)
 Connected to local cluster at 35.200.205.208:9042.
 [cqlsh 5.0.1 | Cassandra 3.9-SNAPSHOT | CQL spec 3.4.2 | Native protocol v4]
 Use HELP for help.
 cqlsh> SHOW HOST
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/clients.md:251]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/_index.md [hunk 20]
```diff
@@ -15,9 +15,9 @@ menu:
...
+As described in the <a href="https://kubernetes.io/docs/setup/best-practices/multiple-zones/">Kubernetes documentation</a>, a single Kubernetes cluster can run only inside a single zone of a single region. This means workloads will be distributed across multiple nodes of a single Kubernetes cluster running in a single region. When geo-distributed SQL workloads like YugabyteDB need to run on Kubernetes, the preferred approach is to create a single YugabyteDB cluster that spans <a href="../multi-zone">multiple zones of a single Kubernetes cluster</a> or multiple geo-distributed Kubernetes clusters. This page documents the latter configuration.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/_index.md:18]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/_index.md [hunk 21]
```diff
@@ -15,9 +15,9 @@ menu:
...
 
-<a href="https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters">Google Kubernetes Engine</a> can be easily configured to support global DNS across multiple Kubernetes clusters. Following instructions highlight how to deploy a 3-region YugabyteDB cluster on 3 Kubernetes clusters, each deployed in a different region. It uses the standard single-zone YugabyteDB Helm Chart to deploy one third of the nodes in the database cluster in each of the 3 clusters.
+<a href="https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters">Google Kubernetes Engine</a> can be configured to support global DNS across multiple Kubernetes clusters. The following instructions show how to deploy a three-region YugabyteDB cluster on three Kubernetes clusters, each deployed in a different region. It uses the standard single-zone YugabyteDB Helm chart to deploy one third of the nodes in the database cluster in each of the three clusters.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/_index.md:20]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md [hunk 22]
```diff
@@ -121,11 +117,13 @@ CURRENT   NAME                                          CLUSTER
           gke_yugabyte_us-west1-b_yugabytedb1           gke_yugabyte_us-west1-b_yugabytedb1
 ```
 
+Note that [Global access](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access) on load balancers is currently in beat and is available only on GKE clusters created using the `rapid` release channel.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md:120]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md [hunk 23]
```diff
@@ -167,29 +165,29 @@ parameters:
...
+kubectl apply -f gke-us-central1-b.yaml --context gke_yugabyte_us-central1-b_yugabytedb2
 ```
 
 ```sh
-$ kubectl apply -f gke-us-east1-b.yaml --context gke_yugabyte_us-east1-b_yugabytedb3
+kubectl apply -f gke-us-east1-b.yaml --context gke_yugabyte_us-east1-b_yugabytedb3
 ```
 
-## 2. Setup global DNS
+## Setup global DNS
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md:182]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md [hunk 24]
```diff
@@ -167,29 +165,29 @@ parameters:
...
+The following YAML file shown adds an internal load balancer (which is not exposed outside its own Google Cloud region) to Kubernetes's built-in `kube-dns` deployment. By default, the `kube-dns` deployment is accessed only by a `ClusterIP` and not a load balancer. You need to allow this load balancer to be [globally accessible](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access) so that each such load balancer is visible to two other load balancers in the other two regions.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md:188]: fixed

[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md:188]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md [hunk 25]
```diff
@@ -395,54 +397,54 @@ gflags:
 
 ### Install YugabyteDB
 
-Now create the overall YugabyteDB cluster in such a way that one third of the nodes are hosted in each Kubernetes cluster.
+Create the YugabyteDB cluste such that one third of the nodes are hosted in each Kubernetes cluster, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md:400]: fixed

File: docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md [hunk 26]
```diff
@@ -454,60 +456,60 @@ yb-tservers          ClusterIP      None            <none>          7100/TCP,900
...
+Follow the instructions provided in [Explore YSQL](../../../../../quick-start/explore/ysql/) and then browse to `http://<external-ip>:7000/tablet-servers` of the YB-Master Admin UI to confirm that tablet peers and their leaders are placed evenly across all three zones for both user data and system data, as per the following illustration:
 
 ![mz-ybtserver](/images/deploy/kubernetes/gke-multicluster-ybtserver.png)
 
-## 7. Connect using external clients
+## Connect using external clients
 
-To connect an external program, get the load balancer `EXTERNAL-IP` address of one of the `yb-tserver-service` service and connect to the 5433 / 9042 ports for YSQL / YCQL services respectively.
+To connect an external program, obtain the load balancer `EXTERNAL-IP` address of one of the `yb-tserver-service` service and connect to the 5433 or 9042 ports for YSQL or YCQL services respectively, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/multi-cluster/gke/helm-chart.md:509]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/aks/helm-chart.md [hunk 27]
```diff
@@ -32,56 +32,56 @@ type: docs
...
   - For more information, see [Installing Helm](https://helm.sh/docs/intro/install/).
 
-- [Microsoft Azure](https://azure.microsoft.com/en-au/pricing/purchase-options/pay-as-you-go/) account
-  - “Pay As You Go” enabled
+- [Microsoft Azure](https://azure.microsoft.com/en-au/pricing/purchase-options/pay-as-you-go/) account with Pay As You Go enabled.
 
 ## Deploy YugabyteDB on an Azure Kubernetes cluster
 
-Follow the steps below to get YugabyteDB up and running on an Azure Kubernetes cluster. The examples below are based on using the macOS.
+The following examples are based on using the macOS.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/aks/helm-chart.md:53]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/aks/helm-chart.md [hunk 28]
```diff
@@ -184,55 +180,49 @@ Merged "yugabytedbAKSCluster" as current context in /Users/yugabyte-user/.kube/c
...
-A browser window appears where you can view the Kubernetes Dashboard:
+A browser window appears where you can view the Kubernetes dashboard, as per the following illustration:
 
 ![Kubernetes Dashboard](/images/deploy/kubernetes/aks/aks-kubernetes-dashboard.png)
 
-### Step 4: Install YugabyteDB using Helm Chart
+### Step 4: Install YugabyteDB using Helm chart
 
-Now that we have our Kubernetes cluster up and running, we'll need to perform the following steps to get YugabyteDB deployed using Helm Chart:
+You need to perform a number of steps to deploy YugabyteDB using the Helm chart:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/aks/helm-chart.md:212]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md [hunk 29]
```diff
@@ -34,67 +34,64 @@ type: docs
     </a>
   </li>
 </ul>
-
 ## Prerequisites
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md:37]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md [hunk 30]
```diff
@@ -34,67 +34,64 @@ type: docs
...
-```
+- Install `kubectl` command line tool by running the following command:
 
-- Configure defaults for gcloud
+  ```sh
+  gcloud components install kubectl
+  ```
 
-Set the project ID as `yugabyte`. You can change this as per your need.
+- Configure defaults for `gcloud` by setting the project ID as `yugabyte`. You can change this as per your need.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md:51]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md [hunk 31]
```diff
@@ -34,67 +34,64 @@ type: docs
...
 
-```sh
-$ gcloud config set project yugabyte
-```
+  ```sh
+  gcloud config set project yugabyte
+  ```
 
-Set the default compute zone as `us-west1-b`. You can change this as per your need.
+- Set the default compute zone as `us-west1-b`. You can change this as per your need.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md:57]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md [hunk 32]
```diff
@@ -107,53 +108,51 @@ Install YugabyteDB in the Kubernetes cluster using the commands desribed in the
...
-$ kubectl create namespace yb-demo
-$ helm install yb-demo yugabytedb/yugabyte --version {{<yb-version version="preview" format="short">}} --namespace yb-demo --wait
+kubectl create namespace yb-demo
+
+helm install yb-demo yugabytedb/yugabyte --version {{<yb-version version="preview" format="short">}} --namespace yb-demo --wait
 ```
 
 #### On Minikube
 
 If you are running in a resource-constrained environment or a local environment, such as Minikube, you have to change the default resource requirements by using the following command:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md:121]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/operator-hub.md [hunk 33]
```diff
@@ -36,83 +36,83 @@ type: docs
...
-NAME                       DISPLAY             VERSION   REPLACES   PHASE
-yugabyte-operator.v0.0.1   Yugabyte Operator   0.0.1                Succeeded
-```
+   ```output
+   NAME                       DISPLAY             VERSION   REPLACES   PHASE
+   yugabyte-operator.v0.0.1   Yugabyte Operator   0.0.1                Succeeded
+   ```
 
-3. Create YugabyteDB Custom Resource to create YugabyteDB cluster using operator deployed above
+3. Create YugabyteDB Custom Resource to create YugabyteDB cluster using the operator deployed in the previous step, as follow:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/operator-hub.md:72]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/operator-hub.md [hunk 34]
```diff
@@ -36,83 +36,83 @@ type: docs
...
 
 ## Configuration flags
 
-For configuration flags, see [Configuration flags](../yugabyte-operator/#configuration-flags).
+For information on configuration flags, see [Configuration flags](../yugabyte-operator/#configuration-flags).
 
 ## Use YugabyteDB
 
-When all of the pods in YugabyteDB cluster are running, you can use the YSQL shell (`ysqlsh`) to access the YSQL API, which is PostgreSQL-compliant.
+When all of the pods in YugabyteDB cluster are running, execute the following command in YSQL shell (`ysqlsh`) to access the YSQL API, which is PostgreSQL-compliant:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/operator-hub.md:100]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md [hunk 35]
```diff
@@ -36,91 +36,97 @@ type: docs
     </a>
   </li>
 </ul>
-
 Create and manage a YugabyteDB cluster with a Kubernetes native custom resource `ybcluster.yugabyte.com`.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md:39]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md [hunk 36]
```diff
@@ -36,91 +36,97 @@ type: docs
...
+## Deploy a YugabyteDB cluster with the operator
 
 To create a YugabyteDB cluster, first you need to register the custom resource that would represent YugabyteDB cluster: `ybclusters.yugabyte.com`.
 
 ```sh
 kubectl create -f deploy/crds/yugabyte.com_ybclusters_crd.yaml
 ```
 
-Setup RBAC for operator and create the operator itself. Run the following command, from root of the repository, to do the same.
+Setup RBAC for operator and create the operator itself. Run the following command, from root of the repository, to do the same:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md:55]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md [hunk 37]
```diff
@@ -36,91 +36,97 @@ type: docs
...
 
 ### Replication factor
 
-Specify the required data replication factor. This is a **required** field.
+Specify the required data replication factor. This is a required field.
 
 ### TLS
 
-Enable TLS encryption for YugabyteDB, if desired. It is disabled by default. You can enable the TLS encryption with 3 flags, explained later. If you have set `enabled` to true, then you need to generate root certificate and key. Specify the two under `rootCA.cert` & `rootCA.key`. Refer to  [TLS encryption docs](../../../../../secure/tls-encryption/server-certificates/) (through the `Generate root configuration` section) for how to generate the certificate and key files.
+Optionally, enable TLS encryption for YugabyteDB. It is disabled by default. You can enable the TLS encryption with three flags. If you have set `enabled` to `true`, then you need to generate the root certificate and key. Specify them under `rootCA.cert` and `rootCA.key`. Refer to [TLS encryption](../../../../../secure/tls-encryption/server-certificates/) for more information.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md:107]: fixed

File: docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md:147]: fixed

**Pull request merged**